tests/tests/test-precompile/test-precompile-staking.ts

```diff
@@ -137,6 +137,32 @@ describeDevMoonbeamAllEthTxTypes("Precompiles - Staking - Join Delegators", (con
     ).to.equal(alith.address, "delegation didn't go through");
     expect(delegatorsAfter.status.toString()).equal("Active");
   });
+
+  it("should have correct delegation amount for ethan to ALITH", async function () {
+    // Check that delegation amount equals MIN_GLMR_STAKING
+    const { result } = await web3EthCall(context.web3, {
+      to: PRECOMPILE_PARACHAIN_STAKING_ADDRESS,
+      data: PARACHAIN_STAKING_INTERFACE.encodeFunctionData("delegationAmount", [
+        ethan.address,
+        alith.address,
+      ]),
+    });
+
+    expect(Number(result)).to.equal(MIN_GLMR_STAKING);
+  });
+
+  it("should have ethan's delegation to ALITH in top delegations", async function () {
+    // Check that delegation is in top delegations
+    const { result } = await web3EthCall(context.web3, {
+      to: PRECOMPILE_PARACHAIN_STAKING_ADDRESS,
+      data: PARACHAIN_STAKING_INTERFACE.encodeFunctionData("isInTopDelegations", [
+        ethan.address,
+        alith.address,
+      ]),
+    });
+
+    expect(Number(result)).to.equal(1);
+  });
```

Comment:
CI error:
```
1) Uncaught error outside test suite:
     Uncaught TypeError: Do not know how to serialize a BigInt
      at stringify (<anonymous>)
      at writeChannelMessage (node:internal/child_process/serialization:159:20)
      at target._send (node:internal/child_process:845:17)
      at target.send (node:internal/child_process:745:19)
      at /moonbeam/tests/node_modules/workerpool/src/worker.js:131:22
```

I have no clue what is wrong, please take a look @crystalin or @librelois

Reply:
There is probably an object being expected and cannot be serialized because it contains a big int (which JSON doesn't support)

Reply:
Usually it is easier to spot using `npm run test-seq` which runs in sequential and doesn't spawn workers for communication